README.md

```diff
@@ -148,9 +151,11 @@ const { data: totalCounter } = useScaffoldContractRead({
 });
 ```
 
+This example retrieves the data returned by the `getGreeting` function of the `YourContract` smart contract. If the function accepts any arguments, they can be passed in the args array. The retrieved data is stored in the `data` property of the returned object.
```

Comment:
Really love that we are mentioning the contract which we are using, should make it a markdown link like so -> [YourContract](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/packages/hardhat/contracts/YourContract.sol) but even current one looks good

Reply:
I actually would vote against not putting a link to the contract because it is for the example and also used in the other hook examples. So, think it would be more clear to show that it can be any contract and not specifically the one we are using. If there is general consensus to have it otherwise happy to make the change though :)

Reply:
>So, think it would be more clear to show that it can be any contract and not specifically the one we are using

Ohh yes agreed and makes sense not to add a link